partcad/src/partcad/actions/part_actions.py

```
@@ -0,0 +1,142 @@
+    "cadquery": "py",
+    "build123d": "py",
+    "scad": "scad",
+    "step": "step",
+    "brep": "brep",
+    "stl": "stl",
+    "3mf": "3mf",
+    "obj": "obj",
+    "gltf": "gltf",
+}
+
+def resolve_enrich_action(project: Project, part_name: str):
```

Ruff (0.8.2)

Missing return type annotation for public function `resolve_enrich_action`

Add return type annotation: `None`

(ANN201)